app/components/recruiter-conversations/conversation/status.js

```diff
@@ -2,16 +2,15 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   profiles: Ember.inject.service(),
-  tagName: 'td',
-  classNames: ['grey'],
-  classNameBindings: ['unreadMessages'],
+  tagName: '',
   unreadMessages: null,
 
 
   onInit: function() {
     const id = this.get('conversation.id');
     return this.get('profiles').getMessages(id)
       .then((bool) => {
+        console.log('unreadmessages:', bool)
```

Comment:
Good point, not very clear whats happening. ` getMessages` makes a request for all the messages for a conversation by an id and returns `true` if there are any unread messages. The original idea was to keep that logic out of the component. I will implement a helper method and rename the function.